src/test/resources/com/puppycrawl/tools/checkstyle/grammar/javadoc/inlinetags/ExpectedLiteralAndCustomInline.txt

```diff
@@ -0,0 +1,175 @@
+|   |--DESCRIPTION -> DESCRIPTION [0:11]
+|   |   `--TEXT ->  @Override [0:11]
+|   `--JAVADOC_INLINE_TAG_END -> } [0:21]
+|--NEWLINE -> \n [0:22]
+|--LEADING_ASTERISK -> * [1:0]
+|--WS ->   [1:1]
+|--JAVADOC_INLINE_TAG -> JAVADOC_INLINE_TAG [1:2]
+|   |--JAVADOC_INLINE_TAG_START -> {@ [1:2]
+|   |--LITERAL -> literal [1:4]
+|   |--DESCRIPTION -> DESCRIPTION [1:11]
+|   |   `--TEXT ->  <p> [1:11]
+|   `--JAVADOC_INLINE_TAG_END -> } [1:15]
+|--NEWLINE -> \n [1:16]
+|--LEADING_ASTERISK -> * [2:0]
+|--WS ->   [2:1]
+|--HTML_ELEMENT -> HTML_ELEMENT [2:2]
+|   |--HTML_TAG_START -> HTML_TAG_START [2:2]
+|   |   |--TAG_OPEN -> < [2:2]
+|   |   |--TAG_NAME -> ul [2:3]
+|   |   `--TAG_CLOSE -> > [2:5]
+|   |--NEWLINE -> \n [2:6]
+|   |--LEADING_ASTERISK -> * [3:0]
+|   |--WS ->   [3:1]
+|   |--HTML_CONTENT -> HTML_CONTENT [3:2]
+|   |   |--HTML_ELEMENT -> HTML_ELEMENT [3:2]
+|   |   |   |--HTML_TAG_START -> HTML_TAG_START [3:2]
+|   |   |   |   |--TAG_OPEN -> < [3:2]
+|   |   |   |   |--TAG_NAME -> li [3:3]
+|   |   |   |   `--TAG_CLOSE -> > [3:5]
+|   |   |   `--HTML_CONTENT -> HTML_CONTENT [3:6]
+|   |   |       `--TEXT ->   [3:6]
+|   |   `--HTML_ELEMENT -> HTML_ELEMENT [3:7]
+|   |       |--HTML_TAG_START -> HTML_TAG_START [3:7]
+|   |       |   |--TAG_OPEN -> < [3:7]
+|   |       |   |--TAG_NAME -> i [3:8]
+|   |       |   `--TAG_CLOSE -> > [3:9]
+|   |       |--HTML_CONTENT -> HTML_CONTENT [3:10]
+|   |       |   `--JAVADOC_INLINE_TAG -> JAVADOC_INLINE_TAG [3:10]
+|   |       |       |--JAVADOC_INLINE_TAG_START -> {@ [3:10]
+|   |       |       |--LITERAL -> literal [3:12]
+|   |       |       |--DESCRIPTION -> DESCRIPTION [3:19]
+|   |       |       |   `--TEXT ->  <crypto_service>.<algorithm_or_type> [3:19]
+|   |       |       `--JAVADOC_INLINE_TAG_END -> } [3:56]
+|   |       `--HTML_TAG_END -> HTML_TAG_END [3:57]
+|   |           |--TAG_OPEN -> < [3:57]
+|   |           |--TAG_SLASH -> / [3:58]
+|   |           |--TAG_NAME -> i [3:59]
+|   |           `--TAG_CLOSE -> > [3:60]
+|   |--NEWLINE -> \n [3:61]
+|   |--LEADING_ASTERISK -> * [4:0]
+|   |--WS ->   [4:1]
+|   `--HTML_TAG_END -> HTML_TAG_END [4:2]
+|       |--TAG_OPEN -> < [4:2]
+|       |--TAG_SLASH -> / [4:3]
+|       |--TAG_NAME -> ul [4:4]
+|       `--TAG_CLOSE -> > [4:6]
+|--NEWLINE -> \n [4:7]
+|--LEADING_ASTERISK -> * [5:0]
+|--WS ->   [5:1]
+|--JAVADOC_INLINE_TAG -> JAVADOC_INLINE_TAG [5:2]
+|   |--JAVADOC_INLINE_TAG_START -> {@ [5:2]
+|   |--LITERAL -> literal [5:4]
+|   |--DESCRIPTION -> DESCRIPTION [5:11]
+|   |   `--TEXT ->  java.sql [5:11]
+|   `--JAVADOC_INLINE_TAG_END -> } [5:20]
+|--NEWLINE -> \n [5:21]
+|--LEADING_ASTERISK -> * [6:0]
+|--WS ->   [6:1]
+|--JAVADOC_INLINE_TAG -> JAVADOC_INLINE_TAG [6:2]
+|   |--JAVADOC_INLINE_TAG_START -> {@ [6:2]
+|   |--LITERAL -> literal [6:4]
+|   |--DESCRIPTION -> DESCRIPTION [6:11]
+|   |   |--TEXT ->   [6:11]
+|   |   |--TEXT -> { [6:12]
+|   |   |--TEXT -> @code code [6:13]
+|   |   `--TEXT -> } [6:23]
```

Comment:
Can we use `more` to collect all of these text nodes, or process them in such a way that we have only one text node?